src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import 'variables';
+@import 'stars';
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.products {
+  display: flex;
+}
+
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
Consider using variables for colors. It will make your code more maintainable and readable. You can add a new variable in the 'variables.scss' file and use it here.